pkg/apis/core/types_shoot.go

```diff
@@ -700,6 +702,19 @@ type Volume struct {
 	Encrypted *bool
 }
 
+// CRI contains information about the Container Runtimes.
+type CRI struct {
+	// The name of the CRI library
+	Name string
+}
+
+// CRIName is a type alias for the CRI name string.
+type CRIName string
+
+const (
+	CRINameContainerD CRIName = "containerd"
```

Comment:
Can we add a doc string for this? Also, I suggest to add `CRINameDocker` as second possible option for the CRI name.

Reply:
Also, in the `v1alpha1` and `v1beta1` version, can you default the `cri.name` field then to `docker` if its not set?

Reply:
Docker shim doesn't implement the CRI.
Actually ContainerD is the CRI for Docker shim.
Maybe it is better to keep this struct empty if no CRI was added.
If CRI section was added currently only "containerd" value is possible.

Reply:
OK, makes sense. Thank you.